### ISSUE TITLE
Zebra additions to http-default-accounts-fingerprints

### ISSUE THREAD

**Issue by nnposter** [nmap/nmap#59]
The attached patch adds fingerprints for [Zebra Technologies](http://www.zebra.com/) web UI to http-default-accounts-fingerprints.lua. Tested on ZTC GK420d with firmware V61.17.5Z.

```
PORT     STATE SERVICE    REASON
8080/tcp open  http-proxy syn-ack ttl 120
| http-default-accounts: [Zebra Printer] credentials found -> :1234 Path:/setgen
|_[Zebra Print Server] credentials found -> admin:1234 Path:/server/TCPIPGEN.htm
```

Patch:
```
--- a/nselib/data/http-default-accounts-fingerprints.lua
+++ b/nselib/data/http-default-accounts-fingerprints.lua
@@ -366,6 +366,48 @@
 })
 
 ---
+--Printers
+---
+table.insert(fingerprints, {
+  name = "Zebra Printer",
+  category = "printer",
+  paths = {
+    {path = "/setgen"}
+  },
+  target_check = function (host, port, path, response)
+    return response.body
+           and response.body:lower():find("<h1>zebra technologies<br>", 1, true)
+  end,
+  login_combos = {
+    {username = "", password = "1234"}
+  },
+  login_check = function (host, port, path, user, pass)
+    local form = {}
+    form["0"] = pass
+    return try_http_post_login(host, port, path, "authorize", "incorrect password", form)
+  end
+})
+
+table.insert(fingerprints, {
+  name = "Zebra Print Server",
+  category = "printer",
+  paths = {
+    {path = "/server/TCPIPGEN.htm"}
+  },
+  target_check = function (host, port, path, response)
+    return http_auth_realm(response) == "Network Print Server"
+           and response.header["server"]
+           and response.header["server"] == "Micro-Web"
+  end,
+  login_combos = {
+    {username = "admin", password = "1234"}
+  },
+  login_check = function (host, port, path, user, pass)
+    return try_http_basic_login(host, port, path, user, pass, false)
+  end
+})
+
+---
 --Remote consoles
 ---
 table.insert(fingerprints, {
```

**Comment by cldrn** [comment]
Hey,

Thanks for the patch. I’ve committed in r 34026.

Cheers.

**Comment by dmiller-nmap** [comment]
@cldrn Thanks for the quick action. For future reference, you can include "Fixes #59" in the commit message to automatically close issue #59.
